beanstalk_test.go

```diff
@@ -1,6 +1,57 @@
 package beanstalk
 
-import "testing"
+import (
+	"net"
+	"testing"
+)
+
+func TestValidURIs(t *testing.T) {
+	l, err := net.Listen("unix", "/tmp/valid.sock")
```

Comment:
To create a test unix domain socket, perhaps create one in a path returned from `os.MkdirTemp()`. That way the test will never fail due to `/tmp/valid.sock` already existing.

Reply:
Updated, using `os.MkdirTemp()`.

Reply:
UPD: as we are not checking the presence of the socket, this code is gone.